plenum/test/pool_transactions/get_txn_request.py

```diff
@@ -0,0 +1,38 @@
+from plenum.common.messages.node_messages import *
+from plenum.test.helper import send_signed_requests, getRepliesFromClientInbox
+from stp_core.loop.eventually import eventually
+from random import randint
+
+
+def testSendGetTxnReqForExistsSeqNo(looper, steward1, stewardWallet):
+    op = {
+        TXN_TYPE: GET_TXN,
+        DATA: 1
+    }
+    req = stewardWallet.signOp(op)
+    send_signed_requests(steward1, [req])
+
+    def chk():
+        receivedReplies = getRepliesFromClientInbox(steward1.inBox,
+                                                    req.reqId)
+        assert len(receivedReplies) == len(steward1.nodeReg)
```

Comment:
Please add a test to check that the returned txn is the one corresponding to requested seq_no.
For example, you can add a new NYM txn, save its seq_no, request it by this seq_no, and check that it's the same NYM.